developing/eid/README.md

```diff
@@ -28,9 +28,16 @@ List of editors that Truss engineers use (sorted alphabetically):
 
 #### vi(m)
 
+[VimAwesome.com](https://vimawesome.com/) provides a list of vim plugins sorted by how many GitHub dotfiles repositories have integrated the plugin.
```

Comment:
this one should probably be a bullet item the same as the others.

Reply:
Hm. Since it's not a plugin, but a list of plugins, I thought it should be separate.

Reply:
maybe having a subheading for _resources_ and one for _good plugins_?

Reply:
I like that, but the heading above this one is "Plugin suggestions". What about something like [this](https://gist.github.com/rpdelaney/e03cef1b2fc7459609df8ff9dd79eea8)?

Reply:
That works for me